app/db/__init__.py

```diff
@@ -0,0 +1,20 @@
+import mysql.connector
+
+class Connection:
+    """This class provides connection to our database"""
+
+    def __init__(self):
+        self.host = "localhost"
```

Comment:
Kindly hide these database config variables either in an `.env` file or inside an `instances` folder

Reply:
done